sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -657,6 +666,7 @@ export class ManagementClient extends LinkEntity {
           operation: Constants.operations.cancelScheduledMessage
         }
       };
+      const associatedLinkName = this._context.sender!.name;
```

Comment:
Same as https://github.com/Azure/azure-sdk-for-js/pull/5638/files#r338762387
`this._context.sender` can be undefined

Reply:
Updated